### PR TITLE
updated design elements

### PR DESCRIPTION
**Pull request by Kilian**
Design updates

todo

- [x] dont say welcome, but explain in title plus add columns
- [x] background for categories/browsers bar and unique checkbox for increased segmentation
- [x] hover/focus effect for tags/browsers, main menu and all links
- [x] fix clear search button in firefox/make style fit other places
- [x] add browser icons to ui like canidevtools
- [x] 320px width horizontal scrollbar
- [x] accidentally apply prettier to the entire style.css

Screenshots:
---

homepage:
image
---

search icon:
image
---

Tip page:
image
---

Checkbox:
image
---

### CODE REVIEW

**Comment by Kilian** [comment]
320px wide no longer has a horizontal scrollbar: 
image

File: src/assets/style.css [hunk 1]
```diff
@@ -113,6 +113,11 @@ body {
 
 a {
   color: var(--accent-color);
+
+  &:hover,
+  &:focus {
+    text-decoration: none;
+  }
```
Comment (captainbrosset) [inline src/assets/style.css:120]: I'm a bit uncomfortable with using CSS nesting so soon. caniuse says 81% of global support. It's hard to trust this % of course, but it's sufficiently low, and the browser market is sufficiently fragmented, that we're bound to have users who don't have support for the feature yet.
I know it's looking pretty green on the usual top 4. But I'm not sure about Opera, Samsung Internet and UCBrowser (caniuse and BCD seem to disagree).

**Comment by Kilian** [comment]
and icons:
image

File: src/assets/style.css (hunk 1 above)
Comment (Kilian) [reply src/assets/style.css]: removed all nesting!

**Comment by captainbrosset** [comment]
Thanks so much for the PR @Kilian. Sorry for the delay here. I'll try to get to this in the coming week.

**Comment by Kilian** [comment]
no rush!

**Comment by captainbrosset** [comment]
Ok I just went through this and really like most of it! Thank you Kilian. So I'll merge the PR and then just push another commit on top of it to change a few minor things.

**Pull request merged**